azext_edge/edge/_help.py

```diff
@@ -471,6 +471,42 @@ def load_iotops_help():
             az iot ops dataflow profile list --in mycluster-ops-instance -g myresourcegroup
     """
 
+    helps[
+        "iot ops dataflow profile create"
+    ] = """
+        type: command
+        short-summary: Create or replace a dataflow profile.
+
+        examples:
+        - name: Create a dataflow profile in the instance 'mycluster-ops-instance' with 2 profile instances.
```

Comment:
We should try to start examples with the most minimum input possible...like in this first example not having --profile-instances (and the example can say will create with default 1).